enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,439 @@
+tracking-link:
+  - "https://issues.redhat.com/browse/OCPSTRAT-619"
+  - "https://issues.redhat.com/browse/WINC-936"
+see-also:
+  - "https://github.com/openshift/enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md"
+  - https://cloud.redhat.com/blog/is-your-operator-air-gap-friendly
+---
+
+# Windows Containers Support in Disconnected Environments
+
+## Release Sign-off Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to support Windows Containers in environments with restricted networks where
+hosts are intentionally impeded from reaching the internet, also known as disconnected or "air-gapped" clusters.
+Currently, Windows nodes configured by the [Windows Machine Config Operator](https://github.com/openshift/windows-machine-config-operator) 
+(WMCO) rely on `containerd``, the OpenShift-managed container runtime, to pull workload images. Also, the WMCO today
+is required to make an external request outside the cluster's internal network to pull the [pause image](https://kubernetes.io/docs/concepts/windows/intro/#pause-container).
+To support disconnected environments, all images need to be pulled from air-gapped mirror registries, whether that be 
+the OpenShift internal image registry or other private registries.
+
+There already exists a protocol for users to publish [registry mirroring configuration](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration),
+namely `ImageDigestMirrorSet` (IDMS), `ImageTagMirrorSet` (ITMS) cluster resources. 
+These are consumed by OpenShift components like the Machine Config Operator (MCO) to apply the settings to Linux
+control-plane and worker nodes, Windows worker nodes do not currently consume or respect mirror registry settings when
+pulling images. This effort will work to plug feature disparity by making the WMCO aware of mirror registry settings at
+operator install time and reactive during its runtime.
+
+
+## Motivation
+
+The motivation here is to expand the Windows containers production use case, enabling users to add Windows nodes and run
+workloads easily and successfully in disconnected clusters. This is an extremely important ask for customer environments
+where Windows nodes must pull images from air-gapped registries for security reasons.
+The added benefit here is that the changes proposed in this enhancement are not restricted to disconnected environments;
+users in __connected__ clusters can take advantage of this mirror registry support. This would allow better performance
```

Comment:
Does mirror registry support imply that customers can configure Windows nodes to pull from other image registries that don't contain the mirror of all the images required for disconnected support?

Reply:
yes, customers would be able to configure other image registries as mirrors as well